metrics/web/lib/common/presentation/app_bar/widget/metrics_app_bar.dart

```diff
@@ -1,15 +1,49 @@
 import 'package:flutter/material.dart';
-import 'package:metrics/common/presentation/strings/common_strings.dart';
+import 'package:metrics/common/presentation/metrics_theme/config/dimensions_config.dart';
 
 /// A common for the metrics application [AppBar] widget.
-class MetricsAppBar extends AppBar {
+class MetricsAppBar extends StatelessWidget {
   /// Creates a [MetricsAppBar].
-  MetricsAppBar({
+  const MetricsAppBar({
     Key key,
-  }) : super(
-          key: key,
-          title: const Text(CommonStrings.metrics),
-          centerTitle: false,
-          elevation: 0.0,
-        );
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: DimensionsConfig.contentWidth,
+      height: DimensionsConfig.appBarHeight,
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: <Widget>[
+          Image.network(
+            'icons/logo-metrics.svg',
+            width: 130.0,
+            height: 32.0,
+            fit: BoxFit.contain,
+          ),
+          Tooltip(
+            message: "Open navigation menu",
```

Comment:
```suggestion
            message: "Open user menu",
```
Should we also move this to the `Strings`?